edx_proctoring/static/proctoring/spec/proctored_exam_spec.js

```diff
@@ -9,7 +9,11 @@ describe('ProctoredExamView', function () {
             'You are taking "' +
             '<a href="<%= exam_url_path %>"> <%= exam_display_name %> </a>' +
             '" as a proctored exam. The timer on the right shows the time remaining in the exam' +
-            '<span id="time_remaining_id" class="pull-right"> <b> </b> </span> </div>' +
+            '<span class="exam-timer-clock"> <span id="time_remaining_id">' +
+            '<b> </b> <button role="button" id="toggle_timer" aria-label="Hide Timer" aria-pressed="false">' +
```

Comment:
Assuming the learner can show the timer once it's hidden, we need a "Show Timer" label as well.

Reply:
@srpearce see: https://github.com/edx/edx-platform/pull/15641#pullrequestreview-56368103